frontend/docker/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ghcr.io/cirruslabs/flutter:3.19.5 AS builder
+
+# Set the working directory
+WORKDIR /app
+
+# Copy the project files into the Docker image
+COPY . /app
+
+# Fetch the dependencies
+RUN flutter pub get
+
+# Build the web application
+RUN flutter build web
```

Comment:
I guess this should be `flutter build web --release`

Reply:
Done